src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class OperationStrategy {

```

Comment:
Firstly, I'd recommend also creating a corresponding interface and implementing it here. Secondly, let's put all strategy and operation handlers' implementations into 'strategy.impl' package instead of 'service.impl'